src/PluginForm/Bluesnap/PaymentMethodAddForm.php

```diff
@@ -0,0 +1,176 @@
+<?php
+
+namespace Drupal\commerce_bluesnap\PluginForm\Bluesnap;
+
+use Drupal\commerce\InlineFormManager;
+use Drupal\commerce_bluesnap\ApiService;
+use Drupal\commerce_payment\PluginForm\PaymentMethodAddForm as BasePaymentMethodAddForm;
+
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Routing\RouteMatchInterface;
+
+use Psr\Log\LoggerInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * The BlueSnap PaymentMethodAddForm.
```

Comment:
**CHANGE REQUEST**

Something more descriptive would be great e.g. "Form for adding a Credit Cart payment method for the Hosted Payment Fields gateway". Not exceeding a single 80-char line as the docblock's header line though.